backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -438,9 +443,31 @@ public Integer getDatabaseVersion() {
 
   @Override
   public int countUsers() {
-    if (!ADMIN.equals(getActiveUser()) && getActiveUser() != null) {
+    if (!ADMIN_USER.equals(getActiveUser()) && getActiveUser() != null) {
       throw new MolgenisException("countUsers denied");
     }
     return MetadataUtils.countUsers(getJooq());
   }
+
+  @Override
+  public String getAdminUserName() {
+    return ADMIN_USER;
+  }
+
+  @Override
+  public String getAdminPasswordDefault() {
+    return (String)
+        EnvironmentProperty.getParameter(
+            Constants.MOLGENIS_ADMIN_PW, ADMIN_PW_DEFAULT, STRING);
+  }
```

Comment:
not sure about making this public, now any caller can access the env param.

Reply:
refactored.